src/AzureRMAssets.ts

```diff
@@ -29,6 +29,14 @@ export class AzureRMAssets {
         return "azuresdk-3.0.0";
     }
 
+    // For test dependency injection only
+    public static setFunctionsMetadata(metadataString: string): void {
```

Comment:
I can't figure out where this is being used?

Reply:
I somehow lost a couple of files. Fixed.